code-challenges.test.js

```diff
@@ -1,51 +1,158 @@
+describe("objectArray", () => { 
+    it("Create a function that takes in an array of objects and returns an array with a sentence about each person with their name capitalized", () => {
+    const people = [
+      { name: "ford prefect", occupation: "a hitchhiker" },
+      { name: "zaphod beeblebrox", occupation: "president of the galaxy" },
+      { name: "arthur dent", occupation: "a radio employee" }
+    ]
+// // //     // Expected output: ["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."]
+    expect(objectArray(people)).toEqual("Ford Prefect is a hitchhiker")
+    expect(objectArray(people)).toEqual("Zaphod Beeblebrox is president of the galaxy")
+    expect(objectArray(people)).toEqual("Arthur Dent is a radio employee.")
+  })
+})
+// // // // FAIL  week-6-assessment-ecostrong/code-challenges.test.js
+// // // ● objectArray › Create a function that takes in an array of objects and returns an array with a sentence about each person with their name capitalized
+// // //   ReferenceError: objectArray is not defined
+
+// // // b) Create the function that makes the test pass.
+
+//Psuedo Code
+//the function will take in an array of objects, the function and returns a sentence about each person with their names capitlaized
+// we will use a higher order function 
+// will use. map to capitalize first letter of all the names
+// will use.split inside of the key name .map. split the objects and capitalize first letter of each word
+//use string interpolation with .map for the name and external .map for the occupation.
+//need to access the key name to extract the values and extract occupation values in order to print job.
+//will have to either split string and capitlize the first and last name of each of the two items in the array by mapping through it
+//each name has a first and last name, so it is an object, will need to split the array in order to map through it. 
+//
+
+const nameArray = (array) => {
+  let name = array.map(value => value.name)
+  let splitName = name.map(value => value.split(" "))
+   console.log(name)
+
+  let capitalizedLetters = nameSplit.map(value => value.map(value => value[0].toUppercase() + value.slice(1)))
+  let combinedName = capitalizedLetters.map(value => value.join(" "))
+  let occupation = array.map(value => value.occupation)
+  let nameOccupation = []
+  for(let i =0; i< array.length; i++){
+    nameOccupation.push(`${joinNames[i]} is a ${occupation[i]}`)
+}
+return nameOccupation.join(" ")
+} 
+console.log(nameArray(people)) 
+
+const nameCapitalized = (array) => {
+  let describe = array.map(value => {
+    let capNames = value.name.split(" ").map(value => value.charAt(0).toUpperCase() + value.slice(1)).join("")
+    return `${capNames} is a ${value.occupation}`
+})
+return describe 
+// console.log(nameCapitalized(people))
+
+// Not passing: I keep getting this syntax error at bottom of the page.
```

Comment:
There are a few things that may be causing this.  Your logic looks excellent for the second function you wrote!  I think for the first function, you may have overthought it a bit, so I am happy to see that you were able to condense it down.  Great work!